sbc_compassion/models/contracts.py

```diff
@@ -71,6 +75,17 @@ def _compute_last_letter(self):
             except TypeError:
                 contract.last_letter = -1
 
+    def _compute_incite_to_write(self):
+
+        if self.last_letter == -1 or self.last_letter < 30:
+            next_birthday = fields.Date.from_string(
+                self.child_id.birthdate).replace(year=date.today.year)
+
+            if next_birthday - date.today() < timedelta(days=0):
+                next_birthday.replace(year=next_birthday.year+1)
+
+            return next_birthday - date.today() > timedelta(days=30)
```

Comment:
the compute method should assign the value to the field instead of returning it. The compute method can also be called on a recordset, so in general a compute method has this signature:
```
def _compute_write_for_birthday_alert_(self):
    for contract in self:
        contract.write_for_birthday_alert = result
```